src/elements/posts-list.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<link rel="import" href="../../bower_components/plastic-image/plastic-image.html">
+
+<link rel="import" href="./shared-styles.html">
+<link rel="import" href="../elements/text-truncate.html">
+
+<dom-module id="posts-list">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment positioning"></style>
+
+    <style>
+
+      :host {
+        display: block;
+      }
+
+      .post {
+        padding: 24px 0;
+        display: block;
+        color: var(--primary-text-color);
+      }
+
+      .post:not(:last-of-type) {
+        border-bottom: 1px dotted var(--divider-color);
+      }
+
+      .image {
+        margin-right: 24px;
+        width: 64px;
+        height: 64px;
+        border-radius: var(--border-radius);
+      }
+
+      .details {
+        height: 100%;
+      }
+
+      .title {
+        line-height: 1.2;
+      }
+
+      .description {
+        margin-top: 8px;
+        color: var(--secondary-text-color)
+      }
+
+      .date {
+        font-size: 12px;
+        text-transform: uppercase;
+        color: var(--secondary-text-color);
+      }
+
+      @media (min-width: 640px) {
+        .image {
+          width: 128px;
+          height: 128px;
+        }
+      }
+
+    </style>
+
+    <template is="dom-repeat" items="[[posts]]" as="post">
+      <a href$="/blog/posts/[[post.id]]/" class="post" layout horizontal>
+        <plastic-image
+          class="image"
+          srcset="[[post.image]]"
+          style$="background-color: [[post.backgroundColor]];"
+          sizing="cover"
+          hidden$="[[!post.image]]"
+          lazy-load preload fade
+        ></plastic-image>
+        <div flex>
+          <div class="details" layout vertical justified>
+            <div>
+              <text-truncate lines="2">
```

Comment:
Nope, it made from the design perspective. I don't want people to make it looking bad